ups/sdm_schemas.table

```diff
@@ -1,4 +0,0 @@
-setupRequired(sconsUtils)
```

Comment:
I think we still need to be able to setup this repo with eups? It's part of lsst_distrib.

Reply:
Ah, okay.  I'll make sure it still "setups" before I merge.

Reply:
@ktlim @ctslater Hmm...  Wonder if instead we should just remove from lsst_distrib?  There is no longer any code here...

Reply:
It was added so that other packages (like ci_hsc) can verify that their products match the schemas here. RFC-672

Reply:
Ah, okay, I see -- thanks!